### ISSUE TITLE
chronometerCountDown going upward only

### ISSUE THREAD

**Issue by aadi-github** [MaikuB/flutter_local_notifications#1863]
Hi, I am actually showing a notification that indicates the remaining time of the Booking of the Vehicle parked at the parking lot. eg. 
From Booking_Time = 2022-10-10 12:25
To Booking_Time = 2022-10-10 18:25

so the problem is, suppose the time remaining is 06:25:00. In the notification it starts properly but no matter what the value for chronometerCountDown = true/false the timer goes upward only. 
if chronometerCountDown = false it shows  06:25:00, 06:25:01, 06:25:02, 06:25:03...this is as expected
if chronometerCountDown = true it shows  -06:25:00, -06:25:01, -06:25:02, -06:25:03...going upward only

Below is my AndroidNOtificationDetails ::
    DateTime whenTimer =
        currentTime.subtract(durationElapsed);
    AndroidNotificationDetails and = AndroidNotificationDetails(
      'parked_vehicle',
      'Vehicle Parking Time Remaining',
      channelDescription:
          'This Notification will notify the user that vehicle\'s time of booking for the parking is about to expire.',
      importance: Importance.max,
      priority: Priority.max,
      channelShowBadge: false,
      ticker: 'parked_vehicle',
      color: Colors.blue,
      onlyAlertOnce: true,
      when: whenTimer.millisecondsSinceEpoch,
      timeoutAfter: whenTimer.difference(DateTime.now()).inMilliseconds,
      usesChronometer: true,
      chronometerCountDown: true, 
      visibility: NotificationVisibility.public,
      ongoing: true,
    );

I hope I have mentioned my problem properly.

NOTE: I have checked the example and it shows same behaviour
when: DateTime.now().millisecondsSinceEpoch - 120 * 1000,
usesChronometer: true,
chronometerCountDown: true, 

In short chronometerCountDown: true only adds the "-" sign nothing else.

**Comment by yallurium** [comment]
Since we're counting down, you need to add to the `when` time, not subtract:
```dart
when: DateTime.now()
            .add(durationElapsed)
            .millisecondsSinceEpoch,
```

**Comment by aadi-github** [comment]
> Since we're counting down, you need to add to the `when` time, not subtract:
> 
> ```dart
> when: DateTime.now()
>             .add(durationElapsed)
>             .millisecondsSinceEpoch,
> ```

Ok now I got it, thank you so much. That was just a silly mistake. I thought whatever the timeStamp I provide, this will start counting Up or Down based on chronometerCountDown value itself. Thanks again and I am marking this as closed now.
